src/Actions/Ticket/UpdateTicket.php

```diff
@@ -35,7 +38,22 @@ public function performAction(): Model
         $ticket->save();
 
         if (is_array($users)) {
-            $ticket->users()->sync($users);
+            $sync = $ticket->users()->sync($users);
+
+            foreach (data_get($sync, 'attached', []) as $user) {
+                CreateEventSubscription::make([
```

Comment:
if validate, then you can catch the "Already subscribed" validation exception, so that there are no duplicate subscriptions.